quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/storage/database/jpa/AbstractJpaConnectionProviderFactory.java

```diff
@@ -110,6 +111,8 @@ protected EntityManager createEntityManager(EntityManagerFactory emf, KeycloakSe
             entityManager = PersistenceExceptionConverter.create(session, emf.createEntityManager());
         }
 
+        entityManager.setFlushMode(FlushModeType.AUTO);
```

Comment:
When I debug[1] it locally `AUTO` mode is already set. As far as I was able to find out `AUTO` is default. Am I missing something?

[1] 
```
cd quarkus
mvn -f server/pom.xml -Dsuspend=true compile quarkus:dev -Dquarkus.args="start-dev"
```